### ISSUE TITLE
Java9 Compatibility

### ISSUE THREAD

**Issue by algobardo** [cs-au-dk/TAJS-Inspector#10]
There are currently 2 issues preventing to update to Java 9:

- The current building on TAJS inspector relies on the plugin `cz.habarta.typescript-generator` that doesn't work properly on Java 9. 
- Current version of gradle (`3.5`) doesn't even recognise version 9 as a valid version.

**Comment by simongregersen** [comment]
Just FYI, as there isn't any development on TAJS inspector at the moment, the `cz.habarta.typescript-generator` doesn't really need to be invoked at every build. It only needs to be run if there are any API changes on the Java-side.

**Comment by algobardo** [comment]
Yeah, but still needs to be disabled https://github.com/cs-au-dk/TAJS-Inspector/blob/java-9-compat/build.gradle
